docs/integrations/cloud/aws-secret-manager.mdx

```diff
@@ -51,13 +51,13 @@ Prerequisites:
     ![access key 2](../../images/integrations/aws/integrations-aws-access-key-2.png)
     ![access key 3](../../images/integrations/aws/integrations-aws-access-key-3.png)
 
-    Navigate to your project's integrations tab in Infisical.
+    1. Navigate to your project's integrations tab in Infisical.
+    2. Click on the **AWS Secrets Manager** tile.
+       ![Select AWS Secrets Manager](../../images/integrations.png)
 
-    ![integrations](../../images/integrations.png)
-
-    Press on the AWS Secrets Manager tile and input your AWS access key ID and secret access key from the previous step.
-
-    ![integration auth](../../images/integrations/aws/integrations-aws-secret-manager-auth.png)
+    3. Select the **Access Key** option for Authentication Mode.
+       ![Select Access Key](../../images/integrations/aws/integrations-aws-secret-manager-auth.png)
+    4. Provide the **access key** and **secret key** for the AWS Iam User.
```

Comment:
If in a different page, there should at least be a link here to forward them to the AWS Assume doc? Maybe in something like a <Tip>

Reply:
Resolved by moving to tab